articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md

```diff
@@ -0,0 +1,485 @@
+
+4. In the **Create virtual network gateway** blade, enter the following information:
+
+   - **Name** - The name of the virtual network gateway.
+
+   - **SKU** - Route-based VPN gateway types are offered in three SKUs: Basic, Standard, and High performance. You must select Standard or High performance if you are creating the network to coexist with an ExpressRoute gateway. You must select High performance SKU to enable active-active mode. You can find more information about SKUs here: [Azure Stack SKUs](https://docs.microsoft.com/en-gb/azure/vpn-gateway/vpn-gateway-about-skus-legacy).
+
+   - **Virtual Network** - This is the virtual network that you created earlier.
+
+   - **Public IP address** - The public IP address to assign to this virtual network gateway. Only dynamic public IP addresses are supported. To create a new public IP address:
+
+      - Under **Public IP address** on the **Create virtual network gateway** blade, select **Create new** on the **Choose public IP address** blade.
+
+      - Enter a name for the public IP address and click **OK**.
+
+   - **Configure BGP ASN** - BGP is the standard routing protocol commonly used on the internet to exchange routing information between two or more networks. BGP enables the Azure VPN Gateways and your on-premises VPN devices, called BGP peers or neighbours, to exchange "routes" that will inform both gateways on the availability and reachability for those prefixes to go through the gateways or routers involved. You should also make sure your on-premises VPN devices support BGP before you enable this feature.
+
+   - **Subscription** - This is your UKCloud for Microsoft Azure subscription.
+
+   - **Resource Group** - The virtual network gateway will be created in the same resource group as the chosen virtual network.
+
+   - **Location** - This will be `frn00006`, which is the location of the Azure Stack.
+
+     ![Create new virtual network gateway](images/azs-browser-create-Vnetwork-gateway.png)
+
+5. Click **Create**.
+
+## Create the local network gateway in Azure Stack
+
+The local network gateway refers to your on-premises network. The following steps outline how to create a local network gateway:
+
+1. In the favourites panel, select **Create a resource**.
+
+   ![Create a resource option in favourites panel](images/azsp_newmenu.png)
+
+2. In the **New** blade, select **Networking**.
+
+   ![Networking option in New blade](images/azs-browser-new-networking.png)
+
+3. In the **Featured** section, select **Local network gateway**.
+
+   ![Create new networking resource](images/azs-browser-networking-create.png)
+
+4. In the **Create local network gateway** blade, enter the following information:
+
+   - **Name** - The name of the local network gateway.
+
+   - **IP address** - This is the public IP address of the VPN device that you want Azure to connect to. Specify a dummy IP address, as this will need to be updated once you have generated the public ip with the connection.
```

Comment:
public IP

Reply:
I'd rephrase this a little to:
This is the public IP address of the VPN device that you want Azure to connect to. Specify a dummy IP address for now as you will generate the public IP later.